frontend/cypress/e2e/Pages/User/UserCreation.test.js

```diff
@@ -0,0 +1,88 @@
+import {
+  Given,
+  When,
+  Then,
+  After
+} from '@badeball/cypress-cucumber-preprocessor'
+
+Given('the IDIR user is on the login page', () => {
+  cy.clearAllCookies()
+  cy.clearAllLocalStorage()
+  cy.clearAllSessionStorage()
+  cy.visit('/')
+  cy.getByDataTest('login-container').should('exist')
+})
+
+When('the IDIR user logs in with valid credentials', () => {
+  cy.loginWith(
+    'idir',
+    Cypress.env('admin_idir_username'),
+    Cypress.env('admin_idir_password')
+  )
+  cy.wait(5000)
```

Comment:
Same as below, can we replace this with a contains or url to speed it up?

Reply:
Sure